src/parser/jobs/sam/modules/Sen.tsx

```diff
@@ -37,7 +37,7 @@ const TSUBAME = [
 	ACTIONS.KAESHI_SETSUGEKKA.id,
 ]
 
-const KENKI_PER_SEN = 5
+const KENKI_PER_SEN = 10
```

Comment:
This will probably mean your support range starts at 5.1 - don't forget to update in the follow up!